src/Backend/OnboardingInsuranceAPI/Areas/Insurance/InsuranceData.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using OnboardingInsuranceAPI.Services;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public record InsuranceData
+{
+    [JsonPropertyName("pid")]
+    public string? Pid { get; set; }
```

Comment:
Is this really nullable? You probably always want to have to which pid this insurance belongs.